src/crypto/tls/handshake_client_tls13.go

```diff
@@ -132,6 +134,7 @@ func (hs *clientHandshakeStateTLS13) handshake() error {
 		return err
 	}
 
+	c.handleEvent(*hs.handshakeTimings)
```

Comment:
I would add this callback to the end of `sendClientFinished()`.

Reply:
I kept it here because KEMTLS (or other things) might want to record more events after sending CFIN. (KEMTLS, for example, sends CFIN before SFIN!) Since the time isn't affected, I think it's fine here?